template/core/data/src/main/java/io/bloco/core/data/models/BookDetailsDto.kt

```diff
@@ -0,0 +1,10 @@
+package io.bloco.core.data.models
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class BookDetailsDto(
+    @SerialName("title")
```

Comment:
I am guessing u did this for consistence, but
**TIP**: if the name is the same as the variable you don't need the annotation :P